web/src/components/common/layouts/NotificationMenu.vue

```diff
@@ -0,0 +1,186 @@
+      <v-btn
+        icon
+        v-bind="props"
+      >
+        <v-badge
+          v-if="totalCountUnreadNotifications > 0"
+          color="error"
+          :content="totalCountUnreadNotifications"
+        >
+          <v-icon
+            size="24"
+            color="white"
+            icon="mdi-bell"
+          />
+        </v-badge>
+        <v-icon
+          v-else
+          size="24"
+          color="white"
+          icon="mdi-bell"
+        />
+      </v-btn>
+    </template>
+    <v-sheet
+      rounded="md"
+      width="390"
+      elevation="11"
+    >
+      <div class="px-8 pb-4 pt-6">
+        <div class="d-flex align-center justify-space-between">
+          <h6 class="text-h5">Notifications</h6>
+          <div>
+            <v-chip
+              :color="hasUnreadNotifications ? 'error' : 'success'"
+              variant="flat"
+              size="small"
+              class="text-white mr-2"
+            >
+              {{ totalCountUnreadNotifications }} Unread
+            </v-chip>
+            <v-chip
+              color="info"
+              variant="flat"
+              size="small"
+              class="text-white"
+            >
+              {{ totalCountTodaysNotifications }} Today
+            </v-chip>
+          </div>
+        </div>
+        <p class="text-subtitle-2 font-weight-regular">
+          {{ latestNotifications.length }} Most Recent
+        </p>
+        <v-divider />
+      </div>
+      <div style="max-height: 400px; overflow-y: scroll">
+        <v-list
+          class="py-0"
+          lines="two"
+        >
+          <v-list-item
+            v-for="notification in latestNotifications"
+            :key="notification.id"
+            :value="notification"
+            class="py-4 px-8"
+            :base-color="notification.isRead ? '' : 'error'"
+            @click="markAsRead(notification)"
+          >
+            <template #append>
+              <v-btn
+                v-if="notification.href"
+                size="x-small"
+                class="mt-0 ml-2 align-top"
+                icon="mdi-link-variant"
+                :color="!notification.isRead ? 'error' : 'success'"
+                @click.stop="openNotification(notification)"
+              />
+            </template>
+            <p class="text-subtitle-1 font-weight-retular font-italic textSecondary">
+              {{ formatDate(notification.createdAt) }}
+            </p>
+            <div>
+              <h6 class="text-subtitle-1 font-weight-semibold mb-1">{{ notification.title }}</h6>
+            </div>
+            <p class="text-subtitle-1 font-weight-regular textSecondary">
+              {{ notification.subtitle }}
+            </p>
+          </v-list-item>
+          <v-divider></v-divider>
+        </v-list>
+      </div>
+      <div class="py-4 px-6 text-center">
+        <v-btn
+          variant="outlined"
+          block
+          :to="{ name: 'NotificationPage' }"
```

Comment:
To follow naming standards I would call this "NotificationsPage" and make the url reflect that. 

Or maybe MyNotificationsPage with the url being /my-notifications, though maybe notifications are always personal, so don't need the "my" prefix since it's implied?